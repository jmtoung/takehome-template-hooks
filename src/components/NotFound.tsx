import React from 'react';
import { Container, Jumbotron } from 'react-bootstrap';

export function NotFound() {
  return (
    <Container>
      <Jumbotron>
        <h1>404! The page you are looking for was not found.</h1>
      </Jumbotron>
    </Container>
  );
}
