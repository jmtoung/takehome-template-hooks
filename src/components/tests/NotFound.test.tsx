import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';
import { NotFound } from '../NotFound';

describe('NotFound', () => {
  it('renders NotFound component correctly.', async () => {
    const wrapper = shallow(<NotFound />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});