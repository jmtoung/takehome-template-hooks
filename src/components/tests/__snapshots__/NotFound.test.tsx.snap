// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders NotFound component correctly. 1`] = `
<Container
  fluid={false}
>
  <Jumbotron
    fluid={false}
  >
    <h1>
      404! The page you are looking for was not found.
    </h1>
  </Jumbotron>
</Container>
`;
